### ISSUE TITLE
How to use mkfsFormatOption? [QUESTION]:

### ISSUE THREAD

**Issue by Rafmoshe2500** [dell/csm#1321]
Hi, I'm interested in creating a storageClass for xfs mount. I want to limit the block size and also the io size. I would happy to understand how to use the parameter: "mkfsFormatOption". I would happy to receive an example of its use.
Thank you.

**Comment by KshitijaKakde** [comment]
Hi @Rafmoshe2500 , you can refer csm-docs to understand on how to use the parameter: "mkfsFormatOption"
https://dell.github.io/csm-docs/docs/csidriver/features/powerflex/#custom-file-system-format-options
Thank you.

**Comment by KshitijaKakde** [comment]
Hi @Rafmoshe2500, you can refer to csm-docs to understand how to use the parameter: "mkfsFormatOption"
https://dell.github.io/csm-docs/docs/csidriver/features/powerflex/#custom-file-system-format-options
Thank you

**Comment by KshitijaKakde** [comment]
Hi @Rafmoshe2500, you can  refer the below examples for use of "mkfsFormatOption"

```
mkfsFormatOption : "-L MyVolume"

mkfsFormatOption: "-L MyVolume -m 1 -T largefile4:
```
mkfsFormatOption is not working for Powerflex , it will get fixed in CSM 1.11
https://github.com/dell/csm/issues/1364
